packages/core/utils/src/ansi-html.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import ansiHTML from 'ansi-html';
+
+export function ansiHtml(ansi: string): string {
```

Comment:
This util is just so we can update it easier later on, `ansi-html` hasn't really been maintained for years but still does the job well, anyway someone might want to replace it or tweak how ansi is transformed into html